R/RunAnalyses.R

```diff
@@ -292,13 +292,14 @@ runCmAnalyses <- function(connectionDetails,
     saveRDS(file.path(outputFolder, "outcomeModelReference.rds"))
 
   # Create cohortMethodData objects -----------------------------
-  subset <- referenceTable[!duplicated(referenceTable$cohortMethodDataFile), ]
-  subset <- subset[subset$cohortMethodDataFile != "", ]
-  subset <- subset[!file.exists(file.path(outputFolder, subset$cohortMethodDataFile)), ]
+  subset <- referenceTable[!duplicated(referenceTable$cohortMethodDataFile),]
+  subset <- subset[subset$cohortMethodDataFile != "",]
+  subset <- subset[!file.exists(file.path(outputFolder, subset$cohortMethodDataFile)),]
   if (nrow(subset) != 0) {
     message("*** Creating cohortMethodData objects ***")
+
     createCmDataTask <- function(i) {
-      refRow <- subset[i, ]
+      refRow <- subset[i,]
```

Comment:
Per our code guidelines, a comma should be followed by a space. Why remove it here?